### PR TITLE
Basic Library Template

### PR DESCRIPTION
**Pull request by having11**
Creates the library template, common PCB, and directory structure

### CODE REVIEW

**Review by nobody5050 — DISMISSED** [review]
`hardware/common PCB/SubZero_Pico_Common/pico_common_arduino/` mixing spaces and not spaces in file path? I would suggest one or the other.

File: hardware/common_PCB/SubZero_Pico_Common/include/Patterns.h [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+ */
+typedef bool (*ExecutePatternCallback)(Adafruit_NeoPixel &strip, uint32_t color,
+                                       uint16_t state, uint16_t ledCount);
+
+enum class PatternType {
+  None = 0,
+  SetAll = 1,
+  Blink = 2,
+  RGBFade = 3,
+  HackerMode = 4
```
Comment (5690Programmers) [inline hardware/common_PCB/SubZero_Pico_Common/include/Patterns.h:20]: 1337 HAXXOR MODE 🕶️ 🚀 🌔

File: robot/src/main/cpp/MultiAxisSubsystem.cpp [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                              {this, _axes})
+      .ToPtr();
+}
+
+frc2::CommandPtr MultiAxisSubsystem::Stop() {
+  auto cmd = frc2::RunCommand([] {}, {}).ToPtr();
+  if (poses.empty()) {
+    return cmd;
+  }
+  for (int i = 0; i < size(_axes); i++) {
```
Comment (having11) [inline robot/src/main/cpp/MultiAxisSubsystem.cpp]: What is `size(_axes)` doing? Is there a global `size` method or does it belong to the `span` itself? You might be thinking of `std::size`, but we should prefer calling it on the container object.

File: robot/src/main/cpp/MultiAxisSubsystem.cpp [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  }
+  for (auto pose : poses) {
+    cmd = std::move(cmd).AlongWith(SetPose(pose))
+  }
+  return cmd;
+}
+
+frc2::CommandPtr MultiAxisSubsystem::Stop(AxisIndex index) {
+  return frc2::InstantCommand([this] { _axes[index]->StopMovement(); },
+                              {this, _axes})
```
Comment (having11) [inline robot/src/main/cpp/MultiAxisSubsystem.cpp]: `_axes` is not a subsystem, it's a `span`

**Pull request merged**